src/pages/SettingsPage.js

```diff
@@ -94,6 +120,29 @@ const SettingsPage = ({
                             Sign Out
                         </Text>
                     </TouchableOpacity>
+                    <View style={[styles.settingsPageBody, styles.mt6]}>
+                        <Text style={[styles.settingsPageHeading, styles.mt2, styles.mb4]} numberOfLines={1}>
+                            Chats
+                        </Text>
+                        <Text style={[styles.textP]}>
+                            How should we display chats on your home screen?
```

Comment:
See my comment above, but I think we could kill this.

Reply:
> > Looking good. If we change the name to 'Chat List Mode', then perhaps we should update named references of 'priority mode' also? But this is minor and def not a blocker
> 
> Ah yeah, good point. I think we should just use "Priority Mode" as the form label then.

I've changed this to:
Priority Mode
Is that what you meant?